components/brave_shields/resources/panel/stories/component-panel.tsx

```diff
@@ -52,7 +54,8 @@ export default {
           adsList: LIST_ADS,
           jsList: LIST_JS,
           httpRedirectsList: [],
-          fingerprintsList: []
+          fingerprintsList: [],
+          faviconUrl: { url: 'https://brave.com/static-assets/images/brave-favicon.png' }
```

Comment:
Can you check this https://github.com/brave/brave-core/pull/11966?
From internal webui, using external url directly is prohibited in security reasion.

Reply:
this state isn't for internal webui. it's part of storybook. cc @petemill

Reply:
Oops! Sorry. then seems fine for me :)